### DISCUSSION TITLE
Is it possible to dynamically update the position of the mesh file in the world?

### DISCUSSION THREAD

**Discussion by neuraljiaang** [NVlabs/curobo#190] [Software Q&A] [answered]
Hello,
We would like to integrate the environment into the world file but some part of our environment is dynamic
Is it possible for us to update the environment of the mesh object with low latency? (might be a duplicate of #178)
Thanks.

**Comment by balakumar-s** [comment, marked as answer]
Yes, you can update pose of an obstacle by calling this function: https://curobo.org/_api/curobo.geom.sdf.world.html#curobo.geom.sdf.world.WorldPrimitiveCollision.update_obstacle_pose

It should be fast as we are not doing any caching w.r.t. obstacle pose.
